docs/eventing/sources/generator/ReadmeTemplate.gomd

```diff
@@ -20,8 +20,11 @@ a class of events from a particular software system. Since different event sourc
 by different Custom Resources, this page provides an index of the available source resource types as
 well as links to installation instructions.
 
-This is a non-exhaustive list of Event sources for Knative.
+For details how to write an Event Source consult one of the following tutorials:
+* [Write an EventSource using ContainerSource](../write-event-source-easy-way)
```

Comment:
I'd update the file name from "write-event-source-easy-way" so that future maintenance is easier, and add an alias to the doc maybe.